barretenberg/cpp/src/barretenberg/proof_system/op_queue/ecc_op_queue.hpp

```diff
@@ -53,11 +68,7 @@ class ECCOpQueue {
     [[nodiscard]] size_t get_previous_size() const { return previous_ultra_ops_size; }
     [[nodiscard]] size_t get_current_size() const { return current_ultra_ops_size; }
 
-    void set_commitment_data(std::array<Point, 4>& commitments)
-    {
-        previous_ultra_ops_commitments = ultra_ops_commitments;
```

Comment:
this doesn't seem to be used anywhere actually, because in the merge protocol we treat what is stored in the `ultra_ops_commitments` (from prev interaction) as `C_T_prev` so I removed